### DISCUSSION TITLE
Export model Error

### DISCUSSION THREAD

**Discussion by bakerwy** [fumitoh/modelx#110] [Q&A]
Previously, I exported very simple model.
This time I wanted to export a standard model that lifelib support but failed.

The error message said as follows.

In [10]: mx.read_model('CashValue_ME').export('CashValue_ME_nomx')

UserWarning: Existing model 'CashValue_ME' renamed to 'CashValue_ME_BAK1'

---------------------------------------------------------------------------

AttributeError                            Traceback (most recent call last)

<ipython-input-10-ea0752eb7b38> in <cell line: 1>()

----> 1 mx.read_model('CashValue_ME').export('CashValue_ME_nomx')

C:\WPy64-31020-20230513\WPy64-31020\python-3.10.2.amd64\lib\site-packages\modelx\core\model.py in __getattr__(self, name)

    445

    446     def __getattr__(self, name):

--> 447         return self._impl.get_attr(name)

    448

    449     def __delattr__(self, name):

C:\WPy64-31020-20230513\WPy64-31020\python-3.10.2.amd64\lib\site-packages\modelx\core\model.py in get_attr(self, name)

    962             return get_interfaces(self.global_refs[name])

    963         else:

--> 964             raise AttributeError(

    965                 "Model '{0}' does not have '{1}'".format(self.name, name)

    966             )

AttributeError: Model 'CashValue_ME' does not have 'export'

it said CashValue_ME doesn't have 'export'

What I have done is very simple.

1. I used latest lifelib, modelx, spyder-modelx. (Old version also has same problem)

2. I changed current directory after checking os.get_cwd(). Specifying model location also failed.

Would you give me some advices?

**Comment by bakerwy** [comment]
What I don't understand is it is working my lab top but it is not working another computer in my company..

**Comment by fumitoh** [comment]
It appears that modelx in the downloadable archive available at [lifelib.io/download.html](https://lifelib.io/download.html) is not the most recent version. To update `modelx`, `spyder-modelx`, and `lifelib` to their latest versions within the downloadable, please follow these steps using the WinPython Command Prompt:

1. Open the WinPython Command Prompt.
2. Run the following commands:

   ```
   pip install -U modelx
   pip install -U spyder-modelx
   pip install -U lifelib
   ```

   These commands will ensure that `modelx`, `spyder-modelx`, and `lifelib` are updated to their latest available versions.

**Reply by bakerwy** [reply]
Okay I will try it again Thanks!
